### DISCUSSION TITLE
Alfred Workflow

### DISCUSSION THREAD

**Discussion by eliemada** [FreeTubeApp/FreeTube#4404] [Q&A]
Good morning, 
I saw a previosu post refering to alfred workflow, but I really would like to build my own, to do so i need to determine if FreeTube supports AppleScript, URL scheme. Does FreeTuve support it ?
Typically what it would do is in the spotlight i would type : `ytb "querry"` and it would search for querry in the freetube searchbar.

Thanks for your possible help

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@PikachuEXE any idea?

**Reply by PikachuEXE** [reply]
I know nothing about AppleScript
I also don't know what "if FreeTube supports AppleScript, URL scheme" means D:

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Our team is unfamiliar with using Alfred Workflows and Applescript. I would recommend contacting Alfred Support for help with workflows

**Comment by dahirsh** [comment]
Came across this post searching for the same solution.

I solved it in Alfred using a custom search with the url: `freetube://https://www.youtube.com/results?search_query={query}`

This feels very hacky to me, but it works for now.
